src/rust/engine/process_execution/src/named_caches.rs

```diff
@@ -54,6 +56,11 @@ impl NamedCaches {
     NamedCaches { local_base }
   }
 
+  // This default suffix is also hard-coded into the Python options code in global_options.py
```

Comment:
Maybe add to `global_options.py` a note about the duplication.

Reply:
It's there already: see the edit I made in that file.